### PR TITLE
Markets list filter notice/pills

### LINKED ISSUE #2373: Trading: Market comments
Market comments

* Disqus like thread below the trading UI

Notes

* Site chat that refreshes every 24 hours
* Market chat that's more like comment section 
* RFQ feature allows users to post an RFQ to chat - refreshes every 24 hours
  * Creators side 
    * Minimum RFQ requirement size (100k) 
    * Minimum trade size (10% = 10k)

    * Choose market 
  * Takers side 
    * Need a order ticket form to make an offer
      * Minimum size (10% etc)
      * Individual price 
  * Interactions
    * Need to be able to respond to offers 
      * No - with comment
      * Yes - triggers fill flow 
  * When both parties accept it triggers the order to fill 

**Problem:**
Description of the problem we're solving with this piece of work. 

* [ ] **Articulate the problem**
  * [ ] Write product doc with product (problems, CUJs, ideas)
    * [ ] Gather any feedback (GH, team, user feedback on Discord, Reddit etc) 
    * [ ] Meeting with any team members who have knowledge on the topic 
  * [ ] Do you have a clear picture of the problem you need to solve? 
* [ ] **Explore different solutions (go wide)**
  * [ ] What are common patterns that solve similar problems? 
  * [ ] Explore 3 x different approaches 
  * [ ] Review with product and dev
  * [ ] Run a user study (optional)
  * [ ] Decide on a direction 
* [ ] **Design solution (start small)**
  * [ ] Scope the solution, what’s in, what’s out
  * [ ] Design the components, flows, states 
  * [ ] Design all breakpoints 
  * [ ] Review with product and dev 
  * [ ] Run a user study (optional) 
  * [ ] Iterate (cycles of design/review) 
  * [ ] Finalise 
* [ ] **Write it up**
  * [ ] Write GitHub tickets for the UI team
    * [ ] Break the piece of work down into logical groupings for dev
    * [ ] Describe components, flows and states 
  * [ ] Walk-through with dev

### LINKED ISSUE #2890: Market list info message bar
https://www.figma.com/file/GKhft5ZOwuH3ANrupGNCLHUU/Market-list-page?node-id=187%3A17189
informs user the invalid market filter is disabled

https://www.figma.com/file/GKhft5ZOwuH3ANrupGNCLHUU/Market-list-page?node-id=187%3A58351
informs user there are markets filtered out. The market count will come from getMarkets getter.

https://www.figma.com/file/GKhft5ZOwuH3ANrupGNCLHUU/Market-list-page?node-id=203%3A32067
informs user no filters are active.

`learn more` links will bring up information modal. text to be written by product and design.

### PR DESCRIPTION
**Pull request by andrewdamelio**
closes #2373, 
closes #2890

### CODE REVIEW

File: packages/augur-ui/src/modules/common/filter-notice.styles.less [hunk 1]
```diff
@@ -0,0 +1,55 @@
+@import (reference) '~assets/styles/shared';
+
+.filterNotice {
+  margin: @size-4 @size-16 @size-24 @size-16;
+
+  > div {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/filter-notice.styles.less:9]: css property should be alphabetic order. we have lint rules for it.

File: packages/augur-ui/src/modules/common/filter-notice.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  content: JSX.Element;
+  show: boolean;
+}
+
+export const FilterNotice = (props: FilterNoticeProps) => {
+  return (
+    <div className={Styles.filterNotice}>
+      {props.show ? (
+        <div>
+          <span className={props.color === 'red' ? Styles.red : Styles.grey}>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/filter-notice.tsx:16]: color should be set in CSS.

File: packages/augur-ui/src/modules/common/filter-tags.styles.less [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+.filterTags {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin: @size-4 @size-16 @size-24 @size-16;
+
+  > div {
+    display: flex;
+    align-items: center;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/filter-tags.styles.less:10]: alphabetic order, super nit pic

File: packages/augur-ui/src/modules/common/form.tsx [hunk 4]
```diff
@@ -1955,7 +1963,7 @@ export const CategoryRow = ({ hasChildren = true, handleClick = noop, active = f
       [Styles.loading]: loading,
       [Styles.disabled]: !hasChildren,
   })}>
-    <span>{category}</span>
+    <span>{category && category.length <= 3 ? category.toUpperCase() : category }</span>
```
Comment (bthaile) [inline packages/augur-ui/src/modules/common/form.tsx:2086]: is `category` already Capitalize case?

File: packages/augur-ui/src/modules/market-cards/market-card.tsx [hunk 5]
```diff
@@ -123,15 +123,19 @@ export default class MarketCard extends React.Component<
...
+      arr.filter(Boolean)
+      .map(label => label.toLowerCase())
+      .map(label => ({
         label,
         onClick: toggleCategory(label, path, history)
       }));
 
     const categoriesWithClick = process(categories[0]);
-    const tagsWithClick = tags.filter(Boolean).map(tag => ({
+    const tagsWithClick = tags.filter(Boolean)
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/market-card.tsx:134]: does "tags" mean something else here? 

V1 "tags" no longer exist. there is a tix to remove them. categories replaced them in the market cards. 
Just FYI, no need to change here.

**Review by bthaile — APPROVED** [review]
lgtm, a few comments and questions

File: packages/augur-ui/src/modules/common/form.tsx (hunk 4 above)
Comment (andrewdamelio) [reply packages/augur-ui/src/modules/common/form.tsx:2086]: we lowercase it currently at the getter level for category selection.

File: packages/augur-ui/src/modules/market-cards/market-card.tsx (hunk 5 above)
Comment (andrewdamelio) [reply packages/augur-ui/src/modules/market-cards/market-card.tsx:134]: correct - this can be removed.

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🙌

 This link will always point to this PR's latest changes: https://peek.run/w9jh9pk

**Pull request merged**